src/state/Store.tsx

```diff
@@ -3,7 +3,7 @@ import { StoreProps } from '../common/interfaces';
 
 const initialState: StoreProps = {
   mapViewFilter: 'rain',
-  isNavOpen: false,
+  isNavOpen: true,
```

Comment:
Hmm, this should force the nav aka sidebar to open on load... is that right? That behaviour doesn't seem to work. Once I click away the intro overlay, the sidebar remains closed.

I personally don't mind that it remains closed, but if the intention was different, maybe this requires another look.

Reply:
No that is intended for the other pages. There is a special check on Home to force this behavior. So it is intended 👌